addons/sale/views/sale_order_views.xml

```diff
@@ -715,7 +710,7 @@
                                 </templates>
                             </kanban>
                         </field>
-                        <div class="float-end d-flex gap-1 mb-2 ms-1"
+                        <div class="float-end d-flex gap-1 my-2"
```

Comment:
Do we need margin below button as [below line](https://github.com/odoo/odoo/pull/167957/files#diff-48cb4309a6006f91b2b40e4c1049860218d419fce1782c7dcc278329803129caR721) already have mt-2? :thinking: